src/graph/graphNodes.ts

```diff
@@ -0,0 +1,128 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { Command } from 'vscode';
+import { DocumentClient } from 'documentdb';
+import { INode } from '../nodes';
+import * as util from "./../util";
+import { GraphViewsManager } from "./GraphViewsManager";
+import { GraphConfiguration } from './GraphConfiguration';
+import { GraphViewServer } from './GraphViewServer';
+
+export class GraphDatabaseNode implements INode {
+	public readonly contextValue: string = "cosmosGraphDatabase";
```

Comment:
Can you add all the basic `OpenInPortal`, `Refresh` commands to this node in package.json?

Should be low-hanging fruit

Reply:
TODO